network/overlay.md

```diff
@@ -241,9 +241,9 @@ When you connect to a published port on any swarm node (whether it is running a
 given service or not), you are redirected to a worker which is running that
 service, transparently. Effectively, Docker acts as a load balancer for your
```

Comment:
This section is confusing because it conflates the concepts of routing mesh, ingress routing mesh, and VIP, and then goes on to say that they can all be turned off by switching to `dns-rr` (which does not make sense in the ingress context).  Clarification would be a follow-up PR.